File: DSA/Anuj_Sharma_590018258/Day1/Question2_LeetCode.c
#include <stdio.h>
 int main(){
    int arr[]={0,1,2,4};
    int n=sizeof(arr)/sizeof(arr[0]);
    int sum= (n*(n+1))/2;
    int add=0;
    for(int i=0;i<n;i++){
        add+=arr[i];
    }
    int m= sum - add;
    printf("The missing number is: %d",m);
    return 0;
 }

File: DSA/MuditaSingh_500122701/Day_1/Question2_Leetcode.java
package DSA.MuditaSingh_500122701.Day1;

public class Question2_Leetcode {
     public static int missingNumber(int[] nums) {
        
        int n=nums.length;
        int sum = (n*(n+1))/2;
        int actual =0;

        for(int num : nums){
            actual += num;
        }

        return sum-actual;
    }

    public static void main(String[] args){
        int [] nums = {0, 1, 2, 4, 5};
        System.out.println("The missing number is: " + missingNumber(nums));
    }
    
}
